Cargo.toml

```diff
@@ -29,6 +29,9 @@ serde = {version = "1", features = ["derive"], optional = true}
 special = "0.10"
 peroxide = { version = "0.32.1" }
 num-traits = "0.2.17"
+rand_xoshiro = { version = "0.6", optional = true, features=["serde1"]}
+itertools = "0.12.1"
+proptest = "1.4.0"
```

Comment:
Is proptest required or can it be a dev dependency?

Reply:
moved :+1: